### PR TITLE
fix(docs): `create_dir_all` doesn't error if path already exists

### PR DESCRIPTION
**Pull request by ThinkChaos**

### CODE REVIEW

**Review by notgull — CHANGES_REQUESTED** [review]
Thanks!

Run `cargo fmt --all` and I will merge this.

**Comment by ThinkChaos** [comment]
Should be good!

Most functions are due for copying the std docs again BTW, but I just went the easy route ;)

File: src/lib.rs [hunk 1]
```diff
@@ -140,15 +140,30 @@ pub async fn create_dir<P: AsRef<Path>>(path: P) -> io::Result<()> {
     unblock(move || std::fs::create_dir(path)).await
 }
 
-/// Creates a directory and its parent directories if they are missing.
+/// Creates a new, empty directory at the provided path
```
Comment (notgull) [inline src/lib.rs:143]: Why was the old above description changed?

**Review by notgull — CHANGES_REQUESTED** [review]
On another look, it looks like this documentation is copied from `create_dir` as opposed to `create_dir_all`. Can you fix this?

File: src/lib.rs (hunk 1 above)
Comment (ThinkChaos) [reply src/lib.rs]: Sorry about that, editing files through GitHub...

File: src/lib.rs [hunk 2]
```diff
@@ -114,48 +114,84 @@ pub async fn copy<P: AsRef<Path>, Q: AsRef<Path>>(src: P, dst: Q) -> io::Result<
     unblock(move || std::fs::copy(&src, &dst)).await
 }
 
-/// Creates a directory.
+/// Creates a new, empty directory at the provided path
```
Comment (notgull) [inline src/lib.rs:117]:
```suggestion
/// Creates a new, empty directory at the provided path.
```

**Review by notgull — APPROVED** [review]
Thanks!

**Pull request merged**